docs/contributing.md

```diff
@@ -0,0 +1,34 @@
+# Contributing to nerdctl
+
+## Before you start actual work
+
+- _look into the issue tracker and PRs_ to see if something similar has been reported or attempted before
+- if _you are fixing a bug, start by opening a bug report_ (if you cannot find one) that explains the problem
+- if _you are developing a new feature, start by opening a feature proposal ticket_ (if you cannot find one)
+that explains what you want to do and why we should do it
+
+Make sure your environment is set-up. Besides go (1.22+) and a working containerd installation, you need
+- golangci-lint
+- goimports-reviser
```

Comment:
This paragraph looks slightly misleading, as these tools are only needed for linting